src/registers.sw

```diff
@@ -0,0 +1,115 @@
+//! Functions to expose the 16 reserved FuelVM registers for ease of use.
+//! Ref: https://github.com/FuelLabs/fuel-specs/blob/master/specs/vm/main.md#semantics
+library registers;
+
+/// Contains zero (0), for convenience.
+pub fn zero() -> u64 {
+    asm() {
+        zero
+    }
+}
+
+/// Contains one (1), for convenience.
+pub fn one() -> u64 {
+    asm() {
+        one
+    }
+}
```

Comment:
Added for completeness, but these don't add any value outside of `asm` blocks, so should probably be cut.